### DISCUSSION TITLE
deep-learning/linear-model/maximum-likelihood-estimation

### DISCUSSION THREAD

**Discussion by giscus** [luweizheng/lulaoshi.info#78] [Announcements]
# deep-learning/linear-model/maximum-likelihood-estimation

最大似然估计就是寻找最优参数，使得观测数据发生的概率最大、统计模型与真实数据最相似。本文是一个入门的最大似然估计教程。

https://lulaoshi.info/deep-learning/linear-model/maximum-likelihood-estimation.html

**Comment by Sgongzi** [comment]
线性回归的假设那里，x服从一个mu sigma正态分布是不是写错了？计量里的线性回归的基本假设X不是随机变量，老师想写的是Y吧。下面的公式好像也有一点问题哦。

**Reply by peacemo** [reply]
我的理解：这里的 $x$ 与线性回归里面的自变量没有关系，而是指代一个服从高斯分布的随机变量。后面也说了，实际上线性回归里面假设的是数据噪声 $\epsilon$ 服从正态分布。
